pkg/skaffold/deploy/kubectl.go

```diff
@@ -93,6 +101,13 @@ func (k *KubectlDeployer) Deploy(ctx context.Context, out io.Writer, builds []bu
 		return errors.Wrap(err, "setting labels in manifests")
 	}
 
+	for _, transform := range manifestTransforms {
```

Comment:
we could theoretically add the same thing to kustomize deployer as well - but it is okay to follow-up with a new PR. 
helm will be a bit trickier.